afs/afs-mapdb-storage/src/main/java/com/powsybl/afs/mapdb/storage/NodeInfoSerializer.java

```diff
@@ -80,6 +81,15 @@ public NodeInfo deserialize(DataInput2 input, int available) throws IOException
         for (int i = 0; i < booleanMetadataSize; i++) {
             metadata.setBoolean(input.readUTF(), input.readBoolean());
         }
-        return new NodeInfo(nodeId, name, pseudoClass, description, creationTime, modificationTime, version, metadata);
+        NodeInfo nodeInfo = new NodeInfo(nodeId, name, pseudoClass, description, creationTime, modificationTime, version, metadata);
+        try {
+            boolean consistent = input.readBoolean();
+            if (consistent) {
+                nodeInfo.consistent();
+            }
+        } catch (IOException e) {
```

Comment:
Should'nt we assume it is consistent by default, for backward compatibility ?

Reply:
The deserialize method should deserialize new nodes but old nodes also (nodes stored without consistent attributes). for old nodes we can assume that they are consistent (to be changed).

Reply:
You should specifically check EOFException instead of IOException to avoid mistake for another problem.